gradebookng/tool/src/java/org/sakaiproject/gradebookng/tool/panels/GradeSummaryTablePanel.java

```diff
@@ -391,15 +391,15 @@ public boolean isVisible() {
 		});
 	}
 
-	private void addInstructorAttributeOrHide(WebMarkupContainer sakaiRubricPreviewButton, Long assignmentId, String studentId, boolean showingStudentView) {
+	private void addInstructorAttributeOrHide(WebMarkupContainer sakaiRubricButtonButton, Long assignmentId, String studentId, boolean showingStudentView) {
```

Comment:
This rename probably should be sakaiRubricButton!

Reply:
Probably ... :)